src/autobot/autobotmodule.cpp

```diff
@@ -37,6 +40,8 @@ std::string AutobotModule::moduleName() const
 void AutobotModule::registerExports()
 {
     framework::ioc()->registerExport<IAutobot>(moduleName(), s_autobot);
+
+    //draw::Painter::extended = AbPaintProvider::instance();
```

Comment:
Better to remove this code or add some TODO

Reply:
In the next PR I will continue to work on this code.